app/controllers/review_controller.rb

```diff
@@ -1,21 +1,35 @@
 class ReviewController < ApplicationController
   def index
     @card = current_user.card_for_review
+    respond_to do |format|
+      if @card.present?
+        format.html
+        format.json { render json: @card, methods: [:image_url],
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.